sdk/core/azure-core-test/inc/azure/core/test/test_context_manager.hpp

```diff
@@ -136,24 +144,37 @@ namespace Azure { namespace Core { namespace Test {
     bool IsLiveMode() const { return TestMode == Azure::Core::Test::TestMode::LIVE; }
 
     constexpr static const char* LiveOnlyToken = "_LIVEONLY_";
+    constexpr static const char* PlaybackOnlyToken = "_RECORDEDONLY_";
```

Comment:
These should match. Is it _PLAYBACKONLY_ or _RECORDEDONLY_.

Reply:
done